### PR TITLE
Remove messaging couch log models

### PR DESCRIPTION
**Pull request by gcapalbo**
(to be merged after all the docs are deleted)

### CODE REVIEW

File: custom/fri/models.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 from dimagi.ext.couchdbkit import *
-from corehq.apps.sms.models import SMSLog, SMS
+from corehq.apps.sms.models import SMS
```
Comment (dimagimon) [inline custom/fri/models.py]: F401 'SMS' imported but unused

**Comment by gcapalbo** [comment]
@calellowitz

**Comment by calellowitz** [comment]
build fail looked unrelated so I kicked it again. 
👍 on ✅ assuming this doc deletion is done?

**Comment by gcapalbo** [comment]
yup, doc deletion is done, should be good to go

**Comment by gcapalbo** [comment]
:white_check_mark:

**Comment by czue** [comment]
:skull: :phone: :email: :tada:

**Pull request merged**